railties/test/generators/action_text_install_generator_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "generators/generators_test_helper"
+require "generators/action_text/install/install_generator"
+
+class ActionText::Generators::InstallGeneratorTest < Rails::Generators::TestCase
+  include GeneratorsTestHelper
+
+  setup do
+    Rails.application = Rails.application.class
+    Rails.application.config.root = Pathname(destination_root)
+  end
+
+  teardown do
+    Rails.application = Rails.application.instance
+  end
+
+  test "creates bin/yarn" do
+    run_generator_instance
+
+    assert_file "bin/yarn"
+  end
+
+  test "installs JavaScript dependencies" do
+    run_generator_instance
+    yarn_commands = @yarn_commands.join("\n")
+
+    assert_match %r"^add .*@rails/actiontext@", yarn_commands
+    assert_match %r"^add .*trix@", yarn_commands
+  end
+
+  test "loads JavaScript dependencies in application.js" do
```

Comment:
A spec which assert warning would be raised when Webpacker is not configured(i.e: `app/javascript/packs/application.js` is missing). We should add it.